engine/runtime/src/test/resources/Test_Caching_Invalidation/src/Main.enso

```diff
@@ -0,0 +1,6 @@
+from Standard.Base import IO
+from project.Atom import all
+
+main =
+    false = 0 == 1
```

Comment:
Why not just `IO.println "hmm..."`?

Reply:
Symbol has to be used IIRC. This particular statement was because I wanted to avoid importing stdlib but in the end that was unavoidable. I can remove the `false` declaration though. Good point.